examples/ios/objc/TableView/TableViewController.m

```diff
@@ -51,8 +51,25 @@ - (void)viewDidLoad
 
     // Set realm notification block
     __weak typeof(self) weakSelf = self;
-    self.notification = [self.array addNotificationBlock:^(RLMResults *data, NSError *error) {
-        [weakSelf.tableView reloadData];
+    self.notification = [self.array addNotificationBlock:^(RLMResults *data, RLMCollectionChange *changes, NSError *error) {
```

Comment:
While the headers and documentation all say `RLMCollectionChange *change`, this one defines `RLMCollectionChange *changes` with `changes` being pluralized.

For consistency, and for more natural sounding English, I think the headers should be changed to say `changes` as well. But that's definitely something we can discuss.